ee/ucp/admin/configure/enable-audit-logging.md

```diff
@@ -1,99 +1,95 @@
 ---
 title: Enable audit logging on UCP
```

Comment:
Suggest removing 'Enable' from the title (as the enabling is just one part of the content offered in the topic).